app/src/main/java/org/mozilla/fenix/settings/wallpaper/WallpaperSettingsFragment.kt

```diff
@@ -0,0 +1,59 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.settings.wallpaper
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.platform.ViewCompositionStrategy
+import androidx.fragment.app.Fragment
+import org.mozilla.fenix.databinding.FragmentWallpaperSettingsBinding
+import org.mozilla.fenix.ext.requireComponents
+import org.mozilla.fenix.theme.FirefoxTheme
+import org.mozilla.fenix.wallpapers.Wallpaper
+
+class WallpaperSettingsFragment : Fragment() {
+    private var _binding: FragmentWallpaperSettingsBinding? = null
+    private val binding get() = _binding!!
+
+    private val wallpaperManager by lazy {
+        requireComponents.wallpaperManager
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        _binding = FragmentWallpaperSettingsBinding.inflate(inflater, container, false)
```

Comment:
Optionally, you might be able to drop the ViewBinding and the XML file for an inline inflation of a ComposeView

Reply:
This might be preferable.

Reply:
Good call, thanks